### PR TITLE
Update site elements to modify contact section with CMS 

### LINKED ISSUE #234: Update contact section with CMS
Contact section includes our mission statement. Offer the option to update that text with the CMS.

#228

### PR DESCRIPTION
**Pull request by reactionic127**
Closes #234

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -19,6 +20,7 @@ const IndexPage = ({ data }) => (
     <Benefits />
     <News data={data.allMarkdownRemark.edges} />
     <Testimonials data={data.allMarkdownRemark.edges} />
+    <Contact data={data.allMarkdownRemark.edges} />
```
Comment (jgwerner) [inline src/pages/index.js]: This section goes below automated marketing.

File: content/contact/mission.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 contentType: contact
 title: Contact
+path: /contact
```
Comment (jgwerner) [inline content/contact/mission.md]: Is adding a path here necessary?

File: src/layouts/index.js [hunk 3]
```diff
@@ -103,7 +102,6 @@ export default class TemplateWrapper extends React.Component {
           <div className="pageContent">{this.props.children()}</div>
           {!this.state.hideNav && (
             <div>
-              <ContactSection title={site.siteMetadata.title} />
               <Footer />
```
Comment (jgwerner) [inline src/layouts/index.js]: @reactionic127 it looks like removing the `Contact` component deletes the info from the internal pages.

Comment (reactionic127) [reply src/layouts/index.js]: Yes you are right. In that case, we should get all frontmatter values in internal layout page also

Comment (reactionic127) [reply src/layouts/index.js]: I will fix it

**Pull request merged**